.github/workflows/test-download.yaml

```diff
@@ -0,0 +1,59 @@
+name: Tests (Download)
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  tests:
+
+    strategy:
+      fail-fast: false  # continue other tests if one test in matrix fails
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+        os: [macos-latest, windows-latest, ubuntu-latest]
```

Comment:
We probably don't need to test this across the whole matrix, I think running it just on ubuntu and the latest version of Node should be fine.